src/app/(afterlogin)/calendar/monthly/page.tsx

```diff
@@ -0,0 +1,134 @@
+'use client';
+
+import BoardTitle from '@/app/_components/common/BoardTitle';
+import Navigation from '@/app/_components/nav/Navigation';
+import ProgressBar from '@/app/_components/tasks/ProgressBar';
+import { ko } from 'date-fns/locale';
+import { format, getDay, parse, startOfWeek } from 'date-fns';
+import { useEffect, useState } from 'react';
+import { Calendar, dateFnsLocalizer } from 'react-big-calendar';
+import '../calendar.css';
+
+interface ScheduleItem {
+  id: number;
+  title: string;
+}
+
+export default function Monthly() {
+  const scheduleTypes: ScheduleItem[] = [
+    { id: 0, title: 'Day' },
+    { id: 1, title: 'Week' },
+    { id: 2, title: 'Month' },
+  ];
+  const [activeScheduleIndex, setActiveScheduleIndex] = useState(0);
+  const [mockData] = useState([
+    {
+      task_id: 1,
+      title: '일어나기',
+      memo: '',
+      start_time: '2025-04-11T09:00:00',
+      end_time: '',
+      address: '경기도 수원시 ...',
+      place_name: 'Cafe ABC',
+      location: { lat: '', lng: '' },
+      is_completed: true,
+    },
+    {
+      task_id: 2,
+      title: 'Team meeting',
+      memo: '주간 회의',
+      start_time: '2025-04-11T15:00:00',
+      end_time: '',
+      address: '경기도 수원시 ...',
+      place_name: 'Zep 4번 룸',
+      location: { lat: '127.1086228', lng: '37.4012191' },
+      is_completed: false,
+    },
+    {
+      task_id: 3,
+      title: '구현하기',
+      memo: '수요일까지 구현해요',
+      start_time: '2025-04-11T13:00:00',
+      end_time: '2025-04-13T14:00:00',
+      address: '경기도 수원시 ...',
+      place_name: 'Cafe DEF',
+      location: { lat: '127.1086228', lng: '37.4012191' },
+      is_completed: false,
+    },
+  ]);
+  const [finishedTaskCount, setFinishedTaskCount] = useState(0);
+  const [calendarMockData] = useState(
+    mockData.map(task => ({
+      ...task,
+      start_time: new Date(task.start_time),
+      end_time: new Date(task.end_time || task.start_time),
+    })),
+  );
+  const locales = { ko };
+
+  const localizer = dateFnsLocalizer({
+    format,
+    parse,
+    startOfWeek,
+    getDay,
+    locales,
+  });
```

Comment:
```suggestion
  const localizer = dateFnsLocalizer({
    format,
    parse,
    startOfWeek,
    getDay,
    locales = { ko },
  });
```
간단하게 작성해도 좋을 것 같습니다! 참고 사항입니다!